bundles/org.openhab.binding.siemensrds/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -169,13 +191,21 @@
 		<label>Energy Savings Level</label>
 		<description>Energy savings level (Green Leaf)</description>
 		<category>qualityofservice</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>QualityOfService</tag>
```

Comment:
Don't think `QualityOfService` applies to 'Energy savings'.

Reply:
Ok. Removed.